### PR TITLE
Magento_Config: avoid using deprecated escape* methods from AbstractB…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Static Tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE

File: app/code/Magento/Config/view/adminhtml/templates/system/config/switcher.phtml [hunk 1]
```diff
@@ -4,34 +4,37 @@
  * See COPYING.txt for license details.
  */
 
-/* @var $block \Magento\Backend\Block\Template */
+/**
+ * @var $block \Magento\Backend\Block\Template
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Config/view/adminhtml/templates/system/config/switcher.phtml]:
```suggestion
/**
 * @var \Magento\Backend\Block\Template $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

File: app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml [hunk 2]
```diff
@@ -4,11 +4,14 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Config\Block\System\Config\Tabs */
+/**
+ * @var $block \Magento\Config\Block\System\Config\Tabs
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml]:
```suggestion
 * @var \Magento\Config\Block\System\Config\Tabs $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml [hunk 3]
```diff
@@ -4,11 +4,14 @@
...
+ */
 ?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml]: It looks like we don't need to have an empty line here
```suggestion
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml]: fixed

File: app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml]: fixed

File: app/code/Magento/Config/view/adminhtml/templates/system/config/switcher.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Config/view/adminhtml/templates/system/config/switcher.phtml]: fixed

File: app/code/Magento/Config/view/adminhtml/templates/system/config/switcher.phtml [hunk 4]
```diff
@@ -4,34 +4,37 @@
  * See COPYING.txt for license details.
  */
 
-/* @var $block \Magento\Backend\Block\Template */
+/**
+ * @var \Magento\Backend\Block\Template $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Config/view/adminhtml/templates/system/config/switcher.phtml]:
```suggestion
/**
 * @var \Magento\Backend\Block\Template $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Config/view/adminhtml/templates/system/config/switcher.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Config/view/adminhtml/templates/system/config/switcher.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Pull request closed without merging**